scripts/core/editor3/helpers/fieldsMeta.ts

```diff
@@ -50,7 +50,9 @@ export function setFieldMetadata(item, fieldKey, contentKey, contentValue) {
 }
 
 export function resetFieldMetadata(item) {
-    Object.keys(item[META_FIELD_NAME]).forEach((key) => {
-        setFieldMetadata(item, key, fieldsMetaKeys.draftjsState, null);
-    });
+    if (item[META_FIELD_NAME]) {
```

Comment:
When does it need this condition? Is it `null` sometimes? When?

Reply:
When this is triggered from multiedit it's failing:
https://github.com/superdesk/superdesk-client-core/pull/3179/files#diff-1718c3ee2990cd8f6be76101964abc0fR326